### ISSUE TITLE
GNOME 44 on Xorg and kitty inconsistent initial window appearance

### ISSUE THREAD

**Issue by marioroy** [kovidgoyal/kitty#6204]
**Describe the bug**
Launching kitty using GNOME 44 on Xorg is a peculiar experience due to mutter-x11-frames regression.

Frames client does not ensure consistent appearance of frames
https://gitlab.gnome.org/GNOME/mutter/-/issues/2673

**To Reproduce**
Must be running GNOME on Xorg. Set the kitty background_opacity to 0.87 or any value for opacity background in ~/.config/kitty/kitty.conf. Launch kitty and subsequently use the keyboard shortcut (shift-ctrl-n) to open many windows.

**Environment details**
```
Fedora Linux 38, GNOME 44, NVIDIA RTX 3070 GPU, 530.41.03 driver.
Or any Linux distribution running GNOME 44.
```

**Comment by marioroy** [comment]
No problem. It is understood. Well, in the event someone mentions it, folks know about the issue coming from mutter-x11-frames. Blessings and grace.

**Comment by marioroy** [comment]
Fyi: The mutter-x11-frames anomaly appears to be resolved for the upcoming mutter 44.1 release.
